aip/general/0154.md

```diff
@@ -78,10 +79,13 @@ message DeleteBookRequest {
   // The current etag of the book.
   // If an etag is provided and does not match the current etag of the book,
   // deletion will be blocked and an ABORTED error will be returned.
-  string etag = 2;
+  string etag = 2 [(google.api.field_behavior) = OPTIONAL];
 }
 ```
 
+In this case, the `etag` field **should** be given a behavior annotation -
```

Comment:
How about:

On a request message, the `etag` ...

This will clarify the difference from the resource itself.

Reply:
Good call, done.